process/request_resources.md

```diff
@@ -0,0 +1,57 @@
+# Requesting Resources
+
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing WG
```

Comment:
I don't think there is a Marketing WG, probably just Marketing Committee (which is just Jenn atm)?

Reply:
```suggestion
## Requests through the OpenSSF Marketing Committee
```